src/config/icons/IconUrl.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useEffect, useState } from 'react';
+import { View } from 'react-native';
+import { SvgUri } from 'react-native-svg';
+import { SvgCss } from 'react-native-svg/css';
+
+import { IconProps, colors, getHitSlops, normalize } from '../../config';
+
+export const IconUrl = ({
+  color = colors.primary,
+  iconName,
+  iconStyle,
+  size = normalize(24),
+  style
+}: IconProps & { iconName: string }) => {
+  // if we can use `SvgUri`, we can render the svg directly per passing an uri
+  const uri = iconName
+    ? iconName.match(/http/)
+      ? iconName
+      : `https://fileserver.smart-village.app/hb-meinquartier/tabler-icons/${iconName}.svg`
```

Comment:
we need to find a solution to reference tabler icons not fixed to a fileserver bucket like hb-meinquartier. can you think of some possibilities?

Reply:
How about moving this url to `secret.js`? And we can add these icons to a fileserver of our own creation, not to the MQGB specific fileserver.

Reply:
Ah yeas, we could have an entry per instance, which can have their own svgs in their fileserver bucket or somewhere else. So the part `https://fileserver.smart-village.app/hb-meinquartier/tabler-icons` should be changed. I will take care of this.

Reply:
there will be a new `icons` settings section with a `svgFolderUrl`, see 5570115